config.schema.json

```diff
@@ -1,182 +1,214 @@
+                                  ]
+                              }
+                          }
+                      },
+                      "motion_properties": {
+                          "title": "Motion Properties",
+                          "type": "object",
+                          "condition": {
+                            "functionBody": "if (model.sensors[arrayIndices] && model.sensors[arrayIndices].sensor_type && model.sensors[arrayIndices].sensor_type === 'Motion Sensor') { return true; } else { return false; };"
+                          },
+                          "properties": {
+                              "value_key": {
+                                  "type": "string",
+                                  "enum": [
+                                      "wind_gust"
+                                  ],
+                                  "condition": {
+                                      "functionBody": "if (!model.local_api && model.sensors[arrayIndices] && model.sensors[arrayIndices].sensor_type && model.sensors[arrayIndices].sensor_type === 'Temperature Sensor') { return true; } else { return false; };"
+                                  }
+                              },
+                              "trigger_value": {
+                                  "type": "number",
+                                  "minimum": 1,
+                                  "description": "At what point (value) to trigger motion detected on/off (1 minimum)."
+                              }
+                          }
+                      },
+                      "occupancy_properties": {
+                          "title": "Occupancy Properties",
+                          "type": "object",
+                          "condition": {
+                            "functionBody": "if (model.sensors[arrayIndices] && model.sensors[arrayIndices].sensor_type && model.sensors[arrayIndices].sensor_type === 'Occupancy Sensor') { return true; } else { return false; };"
+                          },
+                          "properties": {
+                              "value_key": {
+                                  "type": "string",
+                                  "enum": [
+                                      "barometric_pressure",
+                                      "precip",
+                                      "precip_accum_local_day",
+                                      "wind_direction",
+                                      "wind_gust",
+                                      "solar_radiation",
+                                      "uv"
+                                  ],
+                                  "condition": {
+                                      "functionBody": "if (!model.local_api && model.sensors[arrayIndices] && model.sensors[arrayIndices].sensor_type && model.sensors[arrayIndices].sensor_type === 'Temperature Sensor') { return true; } else { return false; };"
```

Comment:
Delete `!model.local_api && ` as `functionBody` applies to both Local and Cloud APIs.